packages/core/admin/admin/src/pages/SettingsPage/pages/ApiTokens/ListView/tests/index.test.js

```diff
@@ -988,7 +990,7 @@ describe('ADMIN | Pages | API TOKENS | ListPage', () => {
   });
 
   it('should not show the create button when the user does not have the rights to create', async () => {
-    useRBAC.mockImplementationOnce(() => ({
+    useRBAC.mockImplementation(() => ({
```

Comment:
Are you sure this doesn't create any new side-effects?

Reply:
what kind of side effects? I am trying to test the different combinations of permissions

Reply:
.mockImplementationOnce this one doesn't work unfortunately

Reply:
Why doesn't it work? Overall I don't mind, if the mocked implementation doesn't affect the next test run. We plan to run more and more tests in parallel and having dependencies / side-effects between them will make this very hard, that's why I am asking.

Reply:
Tbh I don't know but every time I used the .mockImplementationOnce the permissions remain the same as at the beginning, then I search online and I have found that the .mockImplementation works...in addition with the afterAll jest.resetAllMocks(); resolve every possible problem...what do you think?

Reply:
If that is the case I'm fine with it :)